index.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap" rel="stylesheet">
+  <link href="//db.onlinewebfonts.com/c/d14035f6b1afeabafbee4abb2ebf0fc6?family=COCOGOOSE" rel="stylesheet" type="text/css"/>
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet"/>
+  <link rel="stylesheet" href="./style/main.css" />
+  <title>S. Peterson School of Modern Art Exhibition</title>
+</head>
+<body>
+  <section class="top">
```

Comment:
- [ ] [OPTIONAL] it would be better if you change this to a header tag cause it's a header of your app and using the right elements improve the UX and readability

Reply:
There is some misunderstanding. It is the title of web page - over the browser, not inside of page. I already have headers like h1 h3 etc on the page. Could you please check it again?

Reply:
I have change the 1st section tag to header. Hope it helps